lib/madara_orchestrator/mongo_express.star

```diff
@@ -0,0 +1,32 @@
+def apply_default(args):
+    return {"image": "mongo-express", "name": "mongo-express", "port": 8080} | args
+
+
+def run(plan, args, suffix):
+    args = apply_default(args)
+    ports = {}
+    ports["http"] = PortSpec(
+        number=args["port"], application_protocol="http", wait=None
+    )
+
+    name = args["name"] + suffix
+
+    mongo_service = plan.get_service(name="mongodb" + suffix)
+    mongo_port = mongo_service.ports["mongodb"].number
+
+    config = ServiceConfig(
+        image=args["image"],
+        ports=ports,
+        env_vars={
+            "ME_CONFIG_MONGODB_SERVER": mongo_service.name,
+            "ME_CONFIG_MONGODB_PORT": "{}".format(mongo_port),
+            "ME_CONFIG_MONGODB_ENABLE_ADMIN": "True",
```

Comment:
True ?

Reply:
This value should be set to true to enable access to the Mongo Express site.

Reply:
true not True